x-pack/plugins/reporting/server/lib/esqueue/helpers/create_index.js

```diff
@@ -41,6 +41,7 @@ const schema = {
       },
     }
   },
```

Comment:
Not sure if this makes much a difference since we're likely updating an existing index in current installations. Is there more here that I need to follow up with?